app/views/devise/registrations/new.html.slim

```diff
@@ -0,0 +1,36 @@
+section class='auth-form'
+  div class='auth-form__wrapper'
+    div class='auth-form__top'
+      h4 Welcome!
+      h2 Sign up to use the Application
+
+    div class='auth-form__bot'
```

Comment:
```suggestion
    div class='auth-form__header'
      h4 Welcome!
      h2 Sign up to use the Application

    div class='auth-form__body'
```

Should we name this one or other name to make it more meaningful?

Reply:
Agreed updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/6e5051013e49fbd4fc6fe209042953850e62b12d 😎